config/01-ce-auth-defaults.conf

```diff
@@ -68,6 +68,13 @@ AUTH_SSL_SERVER_KEYFILE = /etc/pki/tls/private/localhost.key
 # SciTokens settings
 SCITOKENS_SERVER_AUDIENCE=$(COLLECTOR_HOST)
 
+# Allow READ-level TOKEN authorizations to be generated by the collector.
+SEC_ENABLE_IMPERSONATION_TOKENS=true
+SEC_IMPERSONATION_TOKEN_LIMITS=READ
+
+# Pool password directory for the CE.
+SEC_PASSWORD_DIRECTORY = /etc/condor-ce/passwords.d
```

Comment:
If I understand it correctly, the central collectors and the CEs themselves need this dir. In that case, this should be moved to https://github.com/htcondor/htcondor-ce/blob/master/config/01-common-auth-defaults.conf

Reply:
Done.